validations/management/commands/import_validations.py

```diff
@@ -0,0 +1,223 @@
+
+        # Validate each test case
+        for test_case in test_cases:
+            jsonschema.validate(instance=test_case, schema=schema)
+
+    def _validate_programs_exist(self, test_cases: List[Dict[str, Any]]) -> None:
+        """Validate that all referenced programs exist in the database"""
+        for test_case in test_cases:
+            expected_results = test_case["expected_results"]
+
+            # Handle both single object and array format
+            if isinstance(expected_results, dict):
+                results_list = [expected_results]
+            else:
+                results_list = expected_results
+
+            for result in results_list:
+                program_name = result["program_name"]
+                if not Program.objects.filter(name_abbreviated=program_name).exists():
+                    raise CommandError(
+                        f"Program with name_abbreviated '{program_name}' does not exist. "
+                        f"Test case: {test_case['notes']}"
+                    )
+
+    @transaction.atomic
+    def _process_test_cases(self, test_cases: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+        """Process all test cases and create screens/validations"""
+        results = []
+
+        for test_case in test_cases:
+            result = {
+                "notes": test_case["notes"],
+                "screen_uuid": None,
+                "screen_url": None,
+                "validations_created": [],
+            }
+
+            # Create screen
+            screen = self._create_screen(test_case["household"])
+            result["screen_uuid"] = str(screen.uuid)
+
+            # Generate screen URL
+            white_label = screen.white_label.code
+            frontend_domain = config("FRONTEND_DOMAIN", default="http://localhost:3000")
+            result["screen_url"] = f"{frontend_domain}/{white_label}/{screen.uuid}/results/benefits"
+
+            # Create validations
+            expected_results = test_case["expected_results"]
+            if isinstance(expected_results, dict):
+                results_list = [expected_results]
+            else:
+                results_list = expected_results
+
+            for expected in results_list:
+                validation = self._create_validation(
+                    screen=screen,
+                    program_name=expected["program_name"],
+                    eligible=expected["eligible"],
+                    value=expected.get("value", 0),
+                    notes=test_case["notes"],
+                )
+                result["validations_created"].append(
+                    {
+                        "program_name": validation.program_name,
+                        "eligible": validation.eligible,
+                        "value": str(validation.value),
+                    }
+                )
+
+            results.append(result)
+
+        return results
+
+    def _create_screen(self, household_data: Dict[str, Any]) -> Screen:
```

Comment:
Not necessarily something to act on now but the functionality here seems like it could be used other places (or possibly is similar to existing code)